frontend/src/components/ButtonWithImage/ButtonWithImage.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './ButtonWithImage.scss'
+
+function ButtonWithImage ({ label, icon, labelStyles, onClick }) {
+  return (
+    <div
```

Comment:
Please use `<button>` for buttons. :D Or you integrate the <Button> component in here, which already does the semantic tag handling